### PR TITLE
Implement instantiation.active_record subscriber

### PR DESCRIPTION
**Pull request by ChrisBr**
Partially address https://github.com/influxdata/influxdb-rails/issues/74

https://guides.rubyonrails.org/active_support_instrumentation.html#instantiation-active-record

### CODE REVIEW

File: lib/influxdb/rails/middleware/active_record_subscriber.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        def values(started, finished, payload)
+          {
+            value:        ((finished - started) * 1000).ceil,
+            record_count: payload[:record_count],
+          }
+        end
+
+        def tags(payload)
+          {
+            hook:       "instantiation",
```
Comment (ChrisBr) [inline lib/influxdb/rails/middleware/active_record_subscriber.rb:19]: Not sure if I like this 🤔 Maybe we need to think about differentiate better between AR SQL and AR instantiation.

Comment (hennevogel) [reply lib/influxdb/rails/middleware/active_record_subscriber.rb:19]: You should take this up with AR then, their naming and we should stick to it :smile:

File: lib/influxdb/rails/middleware/active_record_subscriber.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+module InfluxDB
+  module Rails
+    module Middleware
+      class ActiveRecordSubscriber < Subscriber # :nodoc:
+        private
+
+        def values(started, finished, payload)
+          {
+            value:        ((finished - started) * 1000).ceil,
```
Comment (hennevogel) [inline lib/influxdb/rails/middleware/active_record_subscriber.rb:12]: We moved this to `Subscriber.call` now

**Review by hennevogel — CHANGES_REQUESTED** [review]

File: lib/influxdb/rails/middleware/active_record_subscriber.rb (hunk 2 above)
Comment (hennevogel) [reply lib/influxdb/rails/middleware/active_record_subscriber.rb:12]: Ah damn this is in #102 lol

**Comment by hennevogel** [comment]
If you rebase this is good to go

File: lib/influxdb/rails/middleware/active_record_subscriber.rb (hunk 1 above)
Comment (ChrisBr) [reply lib/influxdb/rails/middleware/active_record_subscriber.rb:19]: What I mean is that we have no prefix in our `hook` tag which means it's just `instantiation`.  So what kind of instantiation? Any class? ActiveRecord class? The original hook name is `active_record.instantiation` but we decided at some point to not include the prefix anymore.

Comment (hennevogel) [reply lib/influxdb/rails/middleware/active_record_subscriber.rb:19]: Yes we did, to make our queries cleaner, that's the price we pay.

However, there is only one match for `instantiation` in the ASI docu, so this is fine IMHO.

https://guides.rubyonrails.org/active_support_instrumentation.html

**Review by hennevogel — APPROVED** [review]

**Pull request merged**